styles/_contact.scss

```diff
@@ -10,19 +15,21 @@
     display: flex;
     flex-direction: column;
     align-items: center;
-    padding-top: 200px;
     margin: auto;
+    margin-bottom: 50px;
```

Comment:
Margin too high, it's create an unwanted scroll bar.
Decrease the margin (and verifiy if it works well) **or** delete it if it's **unnecessary**.

Reply:
we can delete it